android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OnMethodCallMapper.kt

```diff
@@ -87,7 +87,7 @@ class OnMethodCallMapper(private var context: Context, private val oneginiMethod
             // Other
             Constants.METHOD_CHANGE_PIN -> startChangePinFlow(result, client)
             Constants.METHOD_GET_APP_TO_WEB_SINGLE_SIGN_ON -> getAppToWebSingleSignOn(call.argument<String>("url"), result, client)
-            Constants.METHOD_GET_USER_PROFILES -> oneginiMethodsWrapper.getUserProfiles(result, client)
+            Constants.METHOD_FETCH_USER_PROFILES -> oneginiMethodsWrapper.fetchUserProfiles(result, client)
```

Comment:
Why is this Fetch, if the rest is called GET?

Reply:
it was called fetch on the dart side and because of this, I thought it is better to leave that name and update the native sides to prevent breaking changes.

Reply:
I think that if we're already making breaking changes we can add this one too as the name fetch implies that it's fetching some resource, which it's not doing. It's only getting it from the sdk. So I think renaming it to GET to on the dart side would make more sense.